pandas/core/apply.py

```diff
@@ -628,7 +628,7 @@ def normalize_dictlike_arg(
 
             cols = Index(list(func.keys())).difference(obj.columns, sort=True)
             if len(cols) > 0:
-                raise KeyError(f"Column(s) {list(cols)} do not exist")
+                raise KeyError(f"Row(s) or Column(s) {list(cols)} do not exist")
```

Comment:
What do you think of `Label(s)` instead of `Row(s) or Column(s)`. This is the generic term that applies to the elements of an Index (whether it be the index or columns of a DataFrame).

Reply:
Thanks, I think `Label(s)` is a better generic term for both `cols` and `columns` too.